### PR TITLE
Render RHS dots menu only on hover or when is open

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Render RHS dots menu only on hover or when is open

### CODE REVIEW

File: components/rhs_comment/rhs_comment.jsx [hunk 1]
```diff
@@ -324,13 +324,13 @@ export default class RhsComment extends React.Component {
         }
 
         let options;
-        if (isEphemeral) {
+        if (isEphemeral && (this.state.hover || this.state.dropdownOpened || this.state.showEmojiPicker)) {
```
Comment (sudheerDev) [inline components/rhs_comment/rhs_comment.jsx]: I think the change here is not needed. `createRemovePostButton` is not expensive as such. We can skip `this.state.dropdownOpened || this.state.showEmojiPicker` though even if we consider skipping initial render

Comment (sudheerDev) [reply components/rhs_comment/rhs_comment.jsx]: Most of the perf was actually because of emoji picker i think

Comment (jespino) [reply components/rhs_comment/rhs_comment.jsx]: The point of havving the `dropdownOpened` and `showEmojiPicker` there is to not "close" the menu when you remove the hovering if the dropdown or the emoji picker is opened.

Comment (jespino) [reply components/rhs_comment/rhs_comment.jsx]: Ohh, sorry, now I understand what you mean. yes, you are right.

Comment (sudheerDev) [reply components/rhs_comment/rhs_comment.jsx]: Do Ephermal posts have emojipicker or dropdowns?

Comment (jespino) [reply components/rhs_comment/rhs_comment.jsx]: My bad, I misunderstood your comment. now I changed it. PTAL.

Comment (sudheerDev) [reply components/rhs_comment/rhs_comment.jsx]: On an unrelated note: This pattern has a flaw wrt to accessibility as we cannot navigate through this actions using keyboard.

Comment (jespino) [reply components/rhs_comment/rhs_comment.jsx]: Yes, but is the same pattern used in the center channel. I think we have to redesign how this works to make it more performant and accessible.

**Pull request merged**